pkg/enforce/enforce.go

```diff
@@ -139,6 +127,34 @@ func EnforceAll(ctx context.Context, ghc ghclients.GhClientsInterface) (EnforceA
 	return enforceAllResults, nil
 }
 
+func RunPoliciesOnInstRepos(ctx context.Context, repos []*github.Repository, ghclient *github.Client, runPolicies RunPoliciesFunc) (
+	EnforceAllResults, string, error) {
+	var instResults = make(EnforceAllResults)
+	var repoLoopErr error
+	var owner string
+	for _, r := range repos {
+		enabled := isBotEnabled(ctx, ghclient, *r.Owner.Login, *r.Name)
+		enforceResults, err := runPolicies(ctx, ghclient, *r.Owner.Login, *r.Name, enabled)
+		if err != nil {
+			// scope of err doesn't extend outside the for loop
+			repoLoopErr = err
+			break
+		}
+		if owner == "" {
+			owner = *r.Owner.Login
+		}
+		for policyName, passed := range enforceResults {
+			if !passed {
+				if instResults[policyName] == nil {
+					instResults[policyName] = make(map[string]int)
+				}
+				instResults[policyName]["totalFailed"] += 1
+			}
+		}
+	}
+	return instResults, owner, repoLoopErr
```

Comment:
Add `config.ClearInstLoc(owner)` here, then remove owner from the return values.